distros/spring/src/test/java/com/connexta/transformation/service/TransformControllerTest.java

```diff
@@ -66,15 +69,64 @@ public void testAcceptedResponseEntity() throws Exception {
                             .finalLocation(new URI(TEST_URI))
                             .metacardLocation(new URI(TEST_URI))))
                 .characterEncoding(StandardCharsets.UTF_8.name()))
-        .andExpect(status().isAccepted());
+        .andExpect(status().isCreated())
+        .andExpect(header().string("Location", "http://localhost/transform/id123"))
```

Comment:
❓ Should the initial part of this expected url be based on what webmvc is defined to specify as opposed to hardcoding it?